pkg/core/root.go

```diff
@@ -42,6 +42,7 @@ func NewKnSourceCommand(knSourceFactory types.KnSourceFactory,
 
 	createCmd := commandFactory.CreateCommand()
 	addCommonFlags(knSourceParams, createCmd)
+	addUpdateFlags(knSourceParams, createCmd)
```

Comment:
why is the name not matching: `addUpdateFlags` and `createCmd`

Reply:
Wondering if we should have one for create even if its implementation is the same?

Reply:
actually, I think the update flags and create flags are same, for most situation.